components/helpers/formatBreed.ts

```diff
@@ -0,0 +1,11 @@
+export const formatBreed = (breed: string) => {
+  const [mainBreed, subBreed] = breed.split("/");
+
+  if (!subBreed) {
+    return breed.charAt(0).toUpperCase() + breed.slice(1);
```

Comment:
2 things:

1. You've got this part of code repeated, remember about the DRY principle,
2. I think such helper function like 'capitalize(text: string)' would be nice to have in helpers

Reply:
good point. next commit is covering it, but suddenly after this branch is merged :hankey: